codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -1649,6 +1826,18 @@ insertTermNames names = do
         ON CONFLICT DO NOTHING
         |]
 
+-- | Insert the given set of type names into the name lookup table
+insertTypeNames :: [NamedRef (Reference.TextReference)] -> Transaction ()
```

Comment:
This is unchanged, it just moved here from the bottom of the file; it got separated from its friends.